services/core-api/app/api/securities/resources/bond.py

```diff
@@ -97,4 +97,55 @@ def put(self, bond_guid):
 
         bond.save()
 
-        return bond
\ No newline at end of file
+        return bond
+
+
+class BondTransferResource(Resource, UserMixin):
+    @api.doc(description='Transfer a bond to a different permit')
+    @requires_role_edit_securities
+    @api.marshal_with(BOND, code=200)
+    def put(self, bond_guid):
+        # Get the bond and validate that it can be transferred
+        bond = Bond.find_by_bond_guid(bond_guid)
+        if bond is None:
+            raise NotFound('No bond was found with the guid provided.')
+        if bond.bond_status_code != "ACT":
+            raise BadRequest('Only active bonds can be transferred.')
+
+        # Get the permit to transfer the bond to and validate it
+        permit_guid = request.json.get('permit_guid', None)
+        if not permit_guid:
+            raise BadRequest('permit_guid is required.')
+        permit = Permit.find_by_permit_guid(permit_guid)
+        if not permit:
+            raise BadRequest('No permit was found with the permit_guid provided.')
+        if permit.permit_guid == bond.permit.permit_guid:
+            raise BadRequest('This bond is already associated with this permit.')
+
```

Comment:
Should validate that the permit_guid belongs to the same mine as the current permit.